src/components/repository-list-item.component.js

```diff
@@ -80,7 +81,9 @@ const renderTitle = repository =>
       />
 
       <Text style={[styles.extraInfoSubject, { paddingTop: 2 }]}>
-        {repository.stargazers_count}
+        {repository.stargazers_count >= 1000
```

Comment:
I think put it in utils will be better.I think  i miss understanding you meanning before.😃😃